sunpy/data/test/let/CNO_lo_sectored_ahead_2015_1hr_level1_11.txt

```diff
@@ -0,0 +1,42 @@
+2015 001.125000 03 00 00 -9999.90 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03
+2015 001.166667 04 00 00 -9999.90 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03
+2015 001.208333 05 00 00 -9999.90 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03
+2015 001.250000 06 00 00 -9999.90 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03
+2015 001.291667 07 00 00 -9999.90 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999   -9999 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03 -9.9999E+03
```

Comment:
I don't know about intentionally, this is the original data file nothing from my side. Although I found some valid data in the same file towards the end of it and I've added those rows skipping the invalid data rows in between.